s4e-web/src/app/app.constants.ts

```diff
@@ -12,6 +13,7 @@ export function s4eConstantsFactory(): IConstants {
   return {
     projection: {toProjection: 'EPSG:3857', coordinates: [19, 52]},
     geoserverUrl: 'http://localhost:8080/geoserver/wms',
+    geoserverWorkspace: 'development',
```

Comment:
And what about cases where it will not be `development` ?

Reply:
For now, it will fail. It should be populated using some configuration returned from backend. I thought to handle it in another issue